docs/content/API-Reference/SQL-API.mdx

```diff
@@ -52,7 +52,7 @@ connection credentials.
 Since the Cube SQL API is Postgres-compatible, please make sure to select **Postgres** as
 a database type when connecting from BI tools.
 
-### Self-hosted Cube
+### <--{"id" : "SQL API"}--> Self-hosted Cube
```

Comment:
Everything seems to work perfectly, except for these three anchors.

They're returning this in the URL for some reason.

```
HOST/docs/backend/sql#undefined-supported-tools
```

Not sure why this is `undefined` here.

Reply:
Here's a link from the deploy preview.

https://deploy-preview-4739--cubejs-docs-staging.netlify.app/backend/sql#undefined-supported-tools

Reply:
When I click the SQL API anchor on that page I get this URL:

https://deploy-preview-4739--cubejs-docs-staging.netlify.app/backend/sql#top

What do you think? Will this will work if you use `"top"` instead of `"SQL API"`?

Reply:
Alternatively, you can change the first three H3 to H2 instead. 🤔

Reply:
I fixed this issue